.github/workflows/verify-k8s.yml

```diff
@@ -176,6 +190,21 @@ jobs:
                 }
           EOF
 
+          # Create secret needed to use fileca
+          cat <<EOF > config/fulcio-secret.yaml
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: fulcio-config
+            namespace: fulcio-dev
+          data:
+            key.pem: $(cat "${{ github.run_id }}-key.pem" | base64)
+            cert.pem: $(cat "${{ github.run_id }}-cert.pem" | base64)
```

Comment:
FYI you can use `stringData` and drop the `base64` (it's k8s magic ✨ )

Reply:
hm. is that going to work alright with a multiline file?

Reply:
It would, but something something yaml 😞